recipes/gff3toembl/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+if [ "${CONDA_PY}" != "27" ]
+then
+    sed -i.bak "s/urllib/urllib.parse/" gff3toembl/EMBLContig.py
```

Comment:
Can we patch this properly using `six` or can this be upstreamed?
Thanks @Cashalow!